### PR TITLE
New README

### PR DESCRIPTION
**Pull request by aaronwolen**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# osfr2
+
+[![Build Status](https://travis-ci.org/aaronwolen/osfr.svg?branch=master)](https://travis-ci.org/aaronwolen/osfr)
+[![Coverage status](https://codecov.io/gh/aaronwolen/osfr/branch/master/graph/badge.svg)](https://codecov.io/github/aaronwolen/osfr?branch=master)
+
+## Overview
+
+osfr provides a suite of functions for interacting with [OSF][osf] that are primarily focused on project management workflows.
+
+OSF (Open Science Framework, <https://osf.io>) is a free and [open source][osf-gh] project management repository designed to support researchers of all technical backgrounds. The service includes unlimited cloud storage and file version history, providing a centralized location for your research materials that can be kept private, shared with select collaborators, or made publicly available with citable DOIs.
```
Comment (csoderberg) [inline README.Rmd]:
```suggestion
OSF (<https://osf.io>) is a free and [open source][osf-gh] project management repository designed to support researchers of all technical backgrounds. The service includes unlimited cloud storage and file version history, providing a centralized location for all your research materials that can be kept private, shared with select collaborators, or made publicly available with citable DOIs.
```

File: README.Rmd [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+This returns an `osf_tbl_node` object with 1 row for the OSF project we retrieved. Let's list the files that have been uploaded to this project:
+
+```{r}
+cr_files <- osf_ls_files(cr_project)
+cr_files
+```
+
+This returns an `osf_tbl_file` with 1 row for each of the uploaded files. We can examine any of these files directly on OSF with `osf_open()`, which opens the corresponding file's view in your default browser.
```
Comment (csoderberg) [inline README.Rmd]:
```suggestion
This returns an `osf_tbl_file` with 1 row for each of the files stored in osf-storage. OSF allows for other resources, like GitHub repos and dropbox folders, to be connected to projects, but currently the osfr package only indexes files stored locally in osf-storage. We can examine any of these files directly on OSF with `osf_open()`, which opens the corresponding file's view in your default browser.
```

File: README.Rmd [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+```{r cleanup, include=FALSE}
+unlink("mtcars.csv")
+osf_rm(my_project, recursive = TRUE, verbose = FALSE)
+```
+
+## Acknowledgments
+
+OSF is developed by the [Center for Open Science][cos] in Charlottesville, VA.
+
+The original version of osfr was developed by @chartgerink. The current version was developed by @aaronwolen and is *heavily* inspired by @jennybc and @lucymcgowan's excellent [googledrive][] package. Seriously, we borrowed a lot of great ideas from them. Other important resources include [http testing](https://ropensci.github.io/http-testing-book/) by Scott Chamberlain and [R Packages](http://r-pkgs.had.co.nz) by Hadley Wickham. Development was also greatly facilitated by OSF's excellent [API documentation][osf-api].
```
Comment (csoderberg) [inline README.Rmd]:
```suggestion
The original version of osfr was developed by @chartgerink and further developed by @bgrich. The current version was developed by @aaronwolen and is *heavily* inspired by @jennybc and @lucymcgowan's excellent [googledrive][] package. Seriously, we borrowed a lot of great ideas from them. Other important resources include [http testing](https://ropensci.github.io/http-testing-book/) by Scott Chamberlain and [R Packages](http://r-pkgs.had.co.nz) by Hadley Wickham. Development was also greatly facilitated by OSF's excellent [API documentation][osf-api].
```

File: README.Rmd [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+2. **files:** this includes both files *and* folders stored on OSF
+3. **users:** individuals with OSF accounts
+
+osfr represents these entities within `osf_tbl`s---specialized tibbles that provide  useful information about the entities like their `name` and unique `id` for users, and API data in the `meta` column that's necessary for osfr's internal functions. Otherwise, they're just `data.frames` and can be manipulated using standard functions from base R or dplyr.
+
+## Usage examples
+
+### Accessing Open Research Materials
+
+Many researchers use OSF to archive and share their work. If you come across a paper that cites an OSF repository, osfr can be used to explore the project and download the associated files---you only need the project's URL or GUID to get started. In this case, we're retrieving the Cancer Reproducibility Project (<https://osf.io/e81xl/>):
```
Comment (csoderberg) [inline README.Rmd]:
```suggestion
Many researchers use OSF to archive and share their work. If you come across a paper that cites an OSF repository, osfr can be used to explore the project and download the associated files---you only need the project's URL or GUID (global unique identifier) to get started. In this case, we're retrieving the Cancer Reproducibility Project (<https://osf.io/e81xl/>):
```

File: README.Rmd (hunk 1 above)
Comment (aaronwolen) [reply README.Rmd]: Is "open science framework" the new Voldemort?  🙂

**Pull request merged**